Build failing on master

```
error[E0308]: mismatched types
   --> evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs:367:21
    |
367 |                     input,
    |                     ^^^^^ expected struct `Vec`, found struct `ethers::prelude::Bytes`
    |
    = note: expected struct `Vec<u8>`
               found struct `ethers::prelude::Bytes`

error[E0308]: mismatched types
   --> evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs:391:37
    |
391 |                 state.set_code(who, code);
    |                                     ^^^^ expected struct `Vec`, found struct `ethers::prelude::Bytes`
    |
    = note: expected struct `Vec<u8>`
               found struct `ethers::prelude::Bytes`

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `foundry-cli v0.1.0 (https://github.com/gakonst/foundry#f770a081)`, intermediate artifacts can be found at `/var/folders/f8/m6308jqs2xdg89n2j44747q80000gn/T/cargo-installGqMI07`
```

--- a/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs
+++ b/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs
@@ -167,6 +167,8 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> SputnikExecutor<CheatcodeStackState<'
             .map(|event| {
                 use HevmConsoleEvents::*;
                 match event {
+                    LogsFilter(inner) => format!("{}", inner.0),
+                    LogBytesFilter(inner) => format!("{}", inner.0),
                     LogNamedAddressFilter(inner) => format!("{}: {:?}", inner.key, inner.val),
                     LogNamedBytes32Filter(inner) => {
                         format!("{}: 0x{}", inner.key, hex::encode(inner.val))

--- a/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs
+++ b/evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs
@@ -364,7 +364,7 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> CheatcodeStackExecutor<'a, 'b, B, P> 
                 let ret = self.call(
                     address,
                     Some(Transfer { source: caller, target: address, value }),
-                    input,
+                    input.to_vec(),
                     target_gas,
                     false,
                     context,
@@ -388,7 +388,7 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> CheatcodeStackExecutor<'a, 'b, B, P> 
             HEVMCalls::Etch(inner) => {
                 let who = inner.0;
                 let code = inner.1;
-                state.set_code(who, code);
+                state.set_code(who, code.to_vec());
             }
         };
 

